src/test/java/com/rocketden/main/UserTests.java

```diff
@@ -126,4 +130,50 @@ public void createNewUserNicknameContainsSpaces() throws Exception {
 
         assertEquals(ERROR.getResponse(), actual);
     }
+
+    @Test
+    public void deleteExistingUser() throws Exception {
+        CreateUserRequest createRequest = new CreateUserRequest();
+        createRequest.setNickname("rocket");
+
+        UserDto expected = new UserDto();
+        expected.setNickname("rocket");
+
+        DeleteUserRequest request = new DeleteUserRequest();
+        request.setNickname("rocket");
+
+        this.mockMvc.perform(post(POST_USER)
```

Comment:
I think we talked about this before, but thoughts on changing this to just `USER` or `USER_URI`? (Same for the tests below.)

Reply:
Whoops I forgot about this! Fixed now.